Tools/autotest/unittest/extract_param_defaults_unittest.py

```diff
@@ -10,7 +10,7 @@
 
 import unittest
```

Comment:
It would be nice if this was `AP_FLAKE8_CLEAN`

Reply:
Both files are AP_FLAKE8_CLEAN and all tests passed